### PR TITLE
Modify dump destination

### LINKED ISSUE #55: Undefined path for the dumps/ folder
Currently, when creating a dump, the dumps folder path is undefined and depends on the system meilisearch is running. `systemd` seems to create it at the root, but when we run meilisearch on a local machine it's in the same directory as MeiliSearch binary. The dumps folder path should be defined by an env variable in the service, and should be consistent among different services :)

### PR DESCRIPTION
**Pull request by alallema**
The folder path for the dump was undefined and was by default at the root.
Modification of the script to set the dump directory in `"/var/opt/meilisearch"`
closed #55

### CODE REVIEW

**Comment by curquiza** [comment]
Thank for starting this change!

Don't forget to update the docs based on what Maryam already added: https://github.com/meilisearch/documentation/pull/1129/files
Point your PR to the `v0.23` branch in the `documentation` repo.

File: scripts/providers/aws/cloud-config.yaml [hunk 1]
```diff
@@ -68,7 +68,7 @@ runcmd:
   - ufw --force enable
   - ufw allow 'Nginx Full'
   - ufw allow 'OpenSSH'
-  - curl https://raw.githubusercontent.com/meilisearch/cloud-scripts/v0.22.0/scripts/deploy-meilisearch.sh | bash -s v0.22.0 aws
+  - curl https://raw.githubusercontent.com/meilisearch/cloud-scripts/dump-dir/scripts/deploy-meilisearch.sh | bash -s dump-dir aws
```
Comment (curquiza) [inline scripts/providers/aws/cloud-config.yaml:71]: Not sure to understand why we change the meilisearch version by `dump-dir`?

Comment (alallema) [reply scripts/providers/aws/cloud-config.yaml:71]: It's for my test ... Sorry I should have put it in draft.

**Review by curquiza — APPROVED** [review]
LGTM!

Also we should use `'` instead of `"` for literal string in our script, I'm going to open an hacktoberfest issue

**Pull request merged**